docs/user/citing.md

```diff
@@ -0,0 +1,66 @@
+If pyDARN contributes to a project that leads to a scientific publication, please acknowledge this fact by citing the software and SuperDARN.  
+
+# How to cite SuperDARN
+
+Super Dual Auroral Radar Network (SuperDARN) is a network of 36 radars 20 institutions, to cite general SuperDARN you can use: 
+
+- Greenwald, R.A., Baker, K.B., Dudeney, J.R. et al. Space Sci Rev (1995) 71: 761. [doi:10.1007/BF00751350](https://doi.org/10.1007/BF00751350)
+
+For the general achievements of the SuperDARN Network, you can read these papers: 
```

Comment:
Recommend "When referencing the general or regional achievements of SuperDARN, we recommend these review papers:"